### ISSUE TITLE
Feature enhancement request:  when ignoring a link, redirect to a page that explains why it is missing.

### ISSUE THREAD

**Issue by wealthychef1** [obsidian-html/obsidian-html#459]
Regarding issue #453.  Instead of just not generating the page in question, would it be possible to have links that are disabled by either limiting the search depth or by specifying the leaf_node tag point to a page that says "This link was disabled by the author" or something to that effect, possibly even allowing me to specify the text?   I'm not sure if this is clear, but I'm just hoping for a better website user experience than a 404 not found error.  :-)  Thanks for all your fast help

**Comment by dwrolvink** [comment]
The best user experience would be to not have any dead links at all of course.

In the case of the `leaf_note` tag, yes, this should be doable. I guess the ideal there would be to select a note in the config.yaml that will be shown when a link is clicked on such a note.

I can pick that one up at some point.

For the max_note_depth case, this is a very complex change. 
In short: we don't really know during the processing of a note whether the note that is linked to will be created (if it is missing) at a later step, and after the conversion we don't really know whether links pointing to missing notes is because of the max_note_depth, or because of wrong links, bugs, etc.

Adding code to the second pass code block, to check for every link whether it exists (in html this time, with html prefix/relative links/etc shenannigans) and then rewriting the link if it doesn't is just a massive amount of work, and a breeding ground for bugs in the future.

So in short I am leaning to a "won't implement" on this one, unless someone thinks of an easier way to implement this.

If you are hosting yourself, then you can always have your nginx or other kind of webserver redirect to a note of your own showing the explanation on a 404. That should be trivial there. If you are not running your own webserver let me know what you do use, maybe there is some trick in that system to make this work that does not involve further stuffing obsidianhtml with rare code :smile:

**Comment by wealthychef1** [comment]
Good idea to do it with the web server.  Chances are that will solve the problem.  Thanks!

**Comment by dwrolvink** [comment]
:+1: I'll close this then, as this is a pretty niche feature request.
